cheat.php

```diff
@@ -644,6 +656,51 @@ function GetCurl( )
 	return $c;
 }
 
+function GetCurlRepository( )
```

Comment:
Way too much code for what it does, You don't need half of it just for the github check. And reduce the timeouts becaus github is not important.

Reply:
What do you want me to remove?

Reply:
Combine it into one function for update checking, remove global var.

You don't need `if` statements that add extra options. Don't need any of the extra headers.

Reply:
Forgot to remove this function?